src/seam-connect/client.ts

```diff
@@ -111,4 +126,51 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientSessionToken(
+    ops: CSTParams
+  ): Promise<APIResponse<ClientSessionResponseInterface>> {
+    let headers: AxiosRequestHeaders = {}
+
+    if (ops.publishedKey?.startsWith("seam_pk")) {
+      // frontend mode
+      headers["seam-publishable-key"] = ops.publishedKey
+    } else if (ops.apiKey?.startsWith("seam_")) {
+      // backend mode
+      headers["seam-api-key"] = ops.apiKey
+    }
+    headers["seam-user-identifier-key"] = ops.userIdentifierKey
+
+    try {
+      const response = await axios.post(
+        ops.endpoint + "internal/client_sessions/create",
```

Comment:
We need to make sure we use the same option paring logic / defaults here. Just tested this locally w/out passing the endpoint and got errors like this

```
r.js:210     POST http://localhost:5173/undefinedinternal/client_sessions/create 404 (Not Found)
dispat
```